api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -40,6 +40,11 @@
    */
   void createWorkspace(String projectName, String workspaceName) throws ApiException;
 
+  /**
+   * Clones an existing workspace.
```

Comment:
What value does this comment add over the function signature?

Reply:
None, removed.